src/components/ebay-textbox/README.md

```diff
@@ -15,8 +15,9 @@ Name | Type | Stateful | Required | Description
 `fluid` | Boolean | No | No |
 `multiline` | Boolean | No | No | renders a multi-line texbox if true
 `invalid` | Boolean | No | No | indicates a field-level error with red border if true
-`icon` | String | No | No | name of the icon from skin
-`icon-position` | String | No | No | Can be "prefix" / "postfix". If not specified or if the value passed is neither "prefix" nor "postfix",  the `icon-position` is set by default to "prefix"
+`prefix-icon` | String | No | No | name of the icon from skin to show as the prefix icon
+`postfix-icon` | String | No | No | name of the icon from skin to show as the postfix icon
+`postfix-aria-label` | String | No | No | aria-label for postfix. Required to be set in order to render postfix button and attach a `textbox-postfix-click event`
```

Comment:
Can we change it from `postfix-aria-label` to `button-aria-label`? And to `textbox-button-click`? With the idea that the button is always at the end, never at the start, of the textbox. We should make combobox have the same language of 'button' too, for consistency.

Reply:
Done